src/Components/Facility/Investigations/Reports/ReportTable.tsx

```diff
@@ -66,14 +69,15 @@ const ReportRow = ({ data, name, min, max }: any) => {
       <TableCell className={className.tableCell} align="right" size="medium">
         {name}
       </TableCell>
-      {data.map((d: any) => {
+      {data.map((d: any, index: number) => {
         const color = getColorIndex({
           min: d?.min,
           max: d?.max,
           value: d?.value,
         });
         return (
           <TableCell
+            key={index}
```

Comment:
Don't use the Array Index as the Key, unless there is no other unique id available.